programmer/tinyprog/__init__.py

```diff
@@ -386,6 +467,60 @@ def boot(self):
             # for the boot command data packet.
             pass
 
+    # similar to program_sectors
+    def verify_sectors(self, addr, data, retry=10):
```

Comment:
This function has  too much copy and pasted code.  Most of the code that was pasted in this function is not used.